code/workspaces/web-app/src/api/dataStorageService.spec.js

```diff
@@ -19,7 +19,7 @@ describe('dataStorageService', () => {
     it('should throw an error if the query fails', () => {
       mockClient.prepareFailure('error');
 
-      return dataStorageService.loadDataStorage().catch((error) => {
+      return dataStorageService.loadDataStorage('project99').catch((error) => {
```

Comment:
Does this have the same problem as the test the other day in that if the error isn't thrown for some reason this will still pass without actually testing the handling of the error?

Reply:
Yes, I think it does - updating...